README.md

```diff
@@ -99,58 +88,6 @@ It's best if you use a public communication channel whenever possible; however,
 
 The troubleshooting guide by Bevy is available [here](https://help.bevylabs.com/article/499-troubleshooting).
 
-## FAQ
```

Comment:
Can we keep the FAQ paragraph here ('## FAQ'), and link to the `faq.md` file under it.